Wikipedia/UI-V5/WMFHomeNearbyCell.h

```diff
@@ -2,11 +2,20 @@
 #import "WMFSaveableTitleCollectionViewCell.h"
 @import CoreLocation;
 
-@interface WMFHomeNearbyCell : WMFSaveableTitleCollectionViewCell
+@class MWKLocationSearchResult;
 
-@property (copy, nonatomic) NSString* descriptionText;
+@interface WMFHomeNearbyCell : WMFSaveableTitleCollectionViewCell
 
-@property (assign, nonatomic) CLLocationDistance distance;
-@property (copy, nonatomic) NSNumber* headingAngle;
+/**
+ *  Display the given search result.
+ *
+ *  Populates the receiver with data in the given result, in addition to binding to properties which are updated
+ *  automatically (i.e. @c distanceFromUser and @c bearingToLocation).
+ *
+ *  @warning Use this instead of @c -setTitle:.
+ *
+ *  @param locationSearchResult The result whose data will be displayed in the receiver.
+ */
+- (void)setLocationSearchResult:(MWKLocationSearchResult*)locationSearchResult withTitle:(MWKTitle*)title;
```

Comment:
> With all the other mutable properties on the search result, the title could easily be included as a mutable property as well so we only need to pass one object /endTroll

I had considered denormalizing the title into the search result as a convenience, but there's no nice way to do it from the fetcher :trollface: 

Do we want to have `ViewModel` models, and rename the `WMFNearbyViewModel` to be a `WMFNearbyViewModelController`?